E3Next/Settings/CharacterSettings.cs

```diff
@@ -133,6 +133,7 @@ public class CharacterSettings : BaseSettings, IBaseSettings
         public List<Spell> HealXTarget = new List<Spell>();
         public List<Spell> HealPets = new List<Spell>();
         public List<Spell> HealOverTime = new List<Spell>();
+        public List<Spell> HealResurrection = new List<Spell>();
```

Comment:
Change this to something more descriptive like `RezSpells` or `ResurrectionSpells`